apps/right-to-rent-check/behaviours/tenants.js

```diff
@@ -144,6 +145,10 @@ module.exports = fields => {
     locals(req, res) {
       const locals = super.locals(req, res);
       locals.tenants = req.sessionModel.get('tenants');
+      _.map(locals.tenants, (tenant) => {
+        tenant['tenant-dob'] = moment(tenant['tenant-dob'], ['YYYY-MM-DD', 'DD-MM-YYYY']).format('DD-MM-YYYY');
```

Comment:
You need to tell moment what format you are expecting if it's not ISO before you can format.  In this situation it can be either of two formats and that's why we have the array.